code/modules/mob/living/simple_animal/slaughter/slaughter.dm

```diff
@@ -67,23 +67,27 @@
 	boost = world.time + 30
 
 
-//////////The Loot
-
+//The loot from killing a slaughter demon - can be consumed to allow the user to blood crawl
 /obj/item/weapon/demonheart
-	name = "demon's heart"
-	desc = "It's still faintly beating with rage"
+	name = "demon heart"
+	desc = "Still it beats furiously, emanating an aura of utter hate."
 	icon = 'icons/obj/surgery.dmi'
-	icon_state = "heart-on"
+	icon_state = "demon_heart"
 	origin_tech = "combat=5;biotech=8"
 
-/obj/item/weapon/demonheart/attack_self(mob/living/user)
-	visible_message("[user] feasts upon the [src].")
+/obj/item/weapon/demonheart/attack(mob/M, mob/living/user, obj/target)
```

Comment:
Probably possible. It might also result in surgically implanting DEMIN HEARTS, so that'd be neat.